apps/storefront/lib/regions.ts

```diff
@@ -23,7 +24,7 @@ export interface Channel {
 }
 
 export const DEFAULT_CHANNEL: Channel = {
-  slug: "default-channel",
+  slug: CHANNEL_SLUG,
   name: "United States Dollar",
   currencyCode: "USD",
```

Comment:
If channel from the env has a different currency, then `name` and `currencyCode` fields will not be valid anymore

Reply:
Additional variables for configuring it might solve the issue